sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -316,6 +316,35 @@ spring:
           useInsecureTrustManager: true
 ```
 
+You can also enable the application to obtain the latest version of the certificate on the portal
+when the inbound certificate is not recognized by using the configuration below.
+
+```yaml
+azure:
+  keyvault:
+    jca:
+      refresh-certificates-when-have-un-trust-certificate: true
+```
+
+Note: If you enable the function of obtaining the latest certificate, your server will be vulnerable
```

Comment:
`enable the function of obtaining the latest certificate` -> `set refresh-certificates-when-have-un-trust-certificate=true`.